charts/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
This file is expected to be inside a chart folder, please move this file to `semantic-hub` folder → https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-02#chart-file-structure

Reply:
done :)